spec/cc/engine/gem_detector_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "spec_helper"
+require "cc/engine/gem_detector"
+
+module CC::Engine
+  describe GemDetector do
+    describe "#present?" do
+      before do
+        @detector = GemDetector.new("./spec/fixtures/Gemfile.lock")
+      end
+
+      it "returns true when spec is present in Gemfile.lock" do
+        @detector.present?("ansi").must_equal(true)
+        @detector.present?("i18n").must_equal(true)
+        @detector.present?("astrolabe").must_equal(true)
```

Comment:
I'm not against the rename, but I'm not sure that predicate-aware syntax is available as we're using minitest. There may be some kind of matcher gem that introduces it but I'd argue it isn't worth it.